trie/committer.go

```diff
@@ -196,22 +170,32 @@ func (c *committer) store(path []byte, n node) node {
 	// usually is leaf node). But small value(less than 32bytes) is not
 	// our target(leaves in account trie only).
 	if hash == nil {
+		// The node is embedded in its parent, in other words, this node
+		// will not be stored in the database independently, mark it as
+		// deleted only if the node was existent in database before.
+		prev, ok := c.tracer.accessList[string(path)]
+		if ok {
+			c.nodes.addNode([]byte(path), &nodeWithPrev{&memoryNode{}, prev})
```

Comment:
[]byte(path) -> no need for casting here ?

Reply:
Fixed